pkg/services/cloudmigration/cloudmigrationimpl/snapshot_mgmt.go

```diff
@@ -225,12 +225,16 @@ func (s *Service) buildSnapshot(ctx context.Context, signedInUser *user.SignedIn
 		}
 	}
 
+	if maxItemsPerPartition == 0 {
```

Comment:
This value comes from a response to a request sent to GMS.

GMS should never actually return 0 so this case shouldn't actually be possible 🤔

Reply:
Fair enough, removed!